core/domain/param_domain_test.py

```diff
@@ -42,46 +43,50 @@ def test_param_spec_validation(self):
         param_spec.obj_type = 'UnicodeString'
         param_spec.validate()
 
-    def test_param_change_validation(self):
+    # TODO: Remove some of the test in the method after the backend is fully type-annotated.
+    # Here ignore[arg-type] and ignore[dict-item] is used to test invalid types.
```

Comment:
Please create an issue and then add the issue number here.
```suggestion
    # TODO(#XXXXX): Remove some of the test in the method after the backend is fully type-annotated.
    # Here ignore[arg-type] and ignore[dict-item] is used to test invalid types.
```